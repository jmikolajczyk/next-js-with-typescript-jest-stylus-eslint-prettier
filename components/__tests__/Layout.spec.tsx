import React from 'react'
import { mount, render } from 'enzyme'
import Layout from '../Layout'

test('hello world', () => {
  const wrapper = mount(<Layout />)
  expect(wrapper.text()).toMatch("I'm here to stay (Footer)")
})

it('renders correctly', () => {
  const tree = render(<Layout />)
  expect(tree).toMatchSnapshot()
})
