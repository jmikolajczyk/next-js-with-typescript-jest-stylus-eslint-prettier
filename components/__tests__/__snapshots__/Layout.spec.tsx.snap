// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <footer>
    <hr />
    <span>
      I'm here to stay (Footer)
    </span>
  </footer>
</div>
`;
